src/dash-table/syntax-tree/lexeme/relational.ts

```diff
@@ -47,16 +51,21 @@ const LEXEME_BASE = {
 
 export const contains: IUnboundedLexeme = R.merge({
     evaluate: relationalEvaluator(([op, exp]) =>
-        typeof op === 'string' &&
-        typeof exp === 'string' &&
-        op.indexOf(exp) !== -1
+        !R.isNil(exp) &&
+        !R.isNil(op) &&
+        (R.type(exp) === 'String' || R.type(op) === 'String') &&
+        op.toString().indexOf(exp.toString()) !== -1
```

Comment:
Reworked the `contains` logic vs. what we had previously discussed. With `operand` disappearing and lhs/rhs always handled with `expression`, it seemed unnatural to have an asymmetric `contains` operator that would do `{a -> 1} contains "1" --> true` but would do `"1" contains {a -> 1} --> false`.

Instead, it now makes sure that both statements are defined and that at least one is a string -- if those requirements are met, it coerces to string if necessary and does the check. The two queries above will now be true.